frontend/packages/ceph-storage-plugin/src/components/create-storage-system/footer.tsx

```diff
@@ -0,0 +1,135 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  WizardFooter,
+  Button,
+  WizardContext,
+  WizardContextType,
+  Alert,
+  AlertActionCloseButton,
+} from '@patternfly/react-core';
+import { k8sCreate } from '@console/internal/module/k8s';
+import { WizardCommonProps } from './reducer';
+import { createSSPayload } from './payloads';
+import { BackingStorageType, StepsId, RHCS } from '../../constants/create-storage-system';
+import { StorageSystemModel } from '../../models';
+import './create-storage-system.scss';
+import { ODF_EXTERNAL_PROVIDERS } from '../../odf-external-providers/external-providers';
+
+const validateBackingStorageStep = (bsType, sc, externalProvider) => {
+  switch (bsType) {
+    case BackingStorageType.EXISTING:
+      return !!sc;
+    case BackingStorageType.EXTERNAL:
+      return !!externalProvider;
+    case BackingStorageType.LOCAL_DEVICES:
+      return true;
+    default:
+      return false;
+  }
+};
+
+const validateStep = (id, state) => {
+  const { storageClass, backingStorage } = state;
+  const { type, externalProvider } = backingStorage;
+  switch (id) {
+    case StepsId.BackingStorage:
+      return validateBackingStorageStep(type, storageClass, externalProvider);
+    case StepsId.CreateLocalVolumeSet:
+    case StepsId.CapacityAndNodes:
+    case StepsId.ConnectionDetails:
```

Comment:
```suggestion
    case StepsId.ConnectionDetails:
```
How are providers supposed to add their validation logic here?

Reply:
Its for RHCS. We dont need to add that.

Reply:
The external vendors only use create storage class step, that only we will be standardizing.